### ISSUE TITLE
Push a new version in maven repo

### ISSUE THREAD

**Issue by sliya** [perwendel/spark-kotlin#22]
The version in the maven repos is not up to date, it would be nice to have the current version in maven central. This would allow me to extend RouteHandler, which is now open but is final in the maven repo

**Comment by perwendel** [comment]
We will push a first release in the coming weeks!

**Comment by jwhitmarsh** [comment]
Any news on this?

**Comment by perwendel** [comment]
Yep, we're wrapping up for a first official release. Some parts left of API to go through but should be before christmas!

**Comment by icholy** [comment]
ping, I need that `exception` handler method. Can you can a `1.0.0-alpha.1` release in in the meantime?
